util/src/main/java/org/datacommons/util/StatChecker.java

```diff
@@ -193,37 +193,40 @@ private boolean isSvObWithNumberValue(McfGraph.PropertyValues node) {
   // Only extract series information for a statVarObservation node that is about a sample place.
   private boolean shouldExtractSeriesInfo(McfGraph.PropertyValues node) {
     String placeDcid = McfUtil.getPropVal(node, Vocabulary.OBSERVATION_ABOUT);
```

Comment:
This private function has one callsite, get the placeDcid there and remove this?

Reply:
done.